Modernise/update the interview project (#1)
2degrees/interview: README.md

# 2degrees interview project

## Code review & live collaboration

We are adding a new model to our application: [`Author`](https://github.com/2degrees/interview/blob/main/src/authn/models.py#L11).

> Here are the Acceptance Tests we received from the product team:

Author: ```suggestion
Here are the requirements from the product team:
```

> Here are the Acceptance Tests we received from the product team:

Author: ```suggestion
Here are the Acceptance Criteria we received from the product team:
```

> Here are the Acceptance Tests we received from the product team:

Author: @mtszsobczak-m2030 feel free to pick one or the other & I'll merge it afterwards.

* We must be able to create Authors
* An Author might have elevated privileges
* An Author must have a display name longer than 3 characters

In [src/authn/models.py](https://github.com/2degrees/interview/blob/main/src/authn/models.py) you'll find the initial 
proposal to implement the above.

* What remarks would you raise during code review?

> * What improvements could you implement, and how would you approach implementing them?

Author: ```suggestion
* What improvements could you implement, and how would you approach them?
```
